sql/_hny_join.sql

```diff
@@ -153,44 +185,120 @@ WITH
         ON r.hny_id = h.hny_id
         WHERE r.one_dev_to_many_hny = 1 AND r.one_hny_to_many_dev = 0
         GROUP BY r.job_number, r.one_dev_to_many_hny, r.one_hny_to_many_dev), 
-    -- b) this would include all other hny devdb relationship 
-    other_relations AS (
-            SELECT
-                r.hny_id,
-                r.job_number::TEXT as job_number,
-                r.all_counted_units::text as classa_hnyaff,
-                r.total_units::text as all_hny_units,
-                r.one_dev_to_many_hny,
-                r.one_hny_to_many_dev,
-                h.project_start_date as project_start_date,
-                h.project_completion_date as project_completion_date,
-                h.extremely_low_income_units::NUMERIC as extremely_low_income_units,
-                h.very_low_income_units::NUMERIC as very_low_income_units,
-                h.low_income_units::NUMERIC as low_income_units,
-                h.moderate_income_units::NUMERIC as moderate_income_units,
-                h.middle_income_units::NUMERIC as middle_income_units,
-                h.other_income_units::NUMERIC as other_income_units,
-                h.studio_units::NUMERIC as studio_units,
-                h."1_br_units"::NUMERIC as "1_br_units",
-                h."2_br_units"::NUMERIC as "2_br_units",
-                h."3_br_units"::NUMERIC as "3_br_units",
-                h."4_br_units"::NUMERIC as "4_br_units",
-                h."5_br_units"::NUMERIC as "5_br_units",
-                h."6_br+_units"::NUMERIC as "6_br+_units",
-                h.unknown_br_units::NUMERIC as unknown_br_units,
-                h.counted_rental_units::NUMERIC as counted_rental_units,
-                h.counted_homeownership_units::NUMERIC as counted_homeownership_units
-            FROM RELATEFLAGS_hny_matches r
-            LEFT JOIN HNY_geo h
-            ON r.hny_id = h.hny_id
-            WHERE NOT (r.one_dev_to_many_hny = 1 AND r.one_hny_to_many_dev = 0))
-        
--- 6) Insert into HNY_devdb  
+
+    -- c) For one hny to many devdb record, pick the m
```

Comment:
Unfinished comment here. These comments are quite important for understanding the code imo